src/ffi/cpython/import.rs

```diff
@@ -0,0 +1,70 @@
+use crate::ffi::{PyInterpreterState, PyObject};
+use std::os::raw::{c_char, c_int, c_uchar};
+
+extern "C" {
+    pub fn _PyImport_IsInitialized(state: *mut PyInterpreterState) -> c_int;
+    // skipped _PyImport_GetModuleId
+    #[cfg(Py_3_7)]
+    pub fn _PyImport_SetModule(name: *mut PyObject, module: *mut PyObject) -> c_int;
+    #[cfg(Py_3_7)]
+    pub fn _PyImport_SetModuleString(name: *const c_char, module: *mut PyObject) -> c_int;
+    pub fn _PyImport_AcquireLock();
+    pub fn _PyImport_ReleaseLock() -> c_int;
+    #[cfg(not(Py_3_7))]
+    pub fn _PyImport_FindBuiltin(name: *const c_char) -> *mut PyObject;
+    #[cfg(all(Py_3_7, not(Py_3_9)))]
+    pub fn _PyImport_FindBuiltin(name: *const c_char, modules: *mut PyObject) -> *mut PyObject;
+    #[cfg(not(Py_3_10))]
+    pub fn _PyImport_FindExtensionObject(a: *mut PyObject, b: *mut PyObject) -> *mut PyObject;
+    #[cfg(not(Py_3_7))]
+    pub fn _PyImport_FixupBuiltin(module: *mut PyObject, name: *const c_char) -> c_int;
+    #[cfg(Py_3_7)]
+    pub fn _PyImport_FixupBuiltin(
+        module: *mut PyObject,
+        name: *const c_char,
+        modules: *mut PyObject,
+    ) -> c_int;
+    #[cfg(not(Py_3_7))]
+    pub fn _PyImport_FixupExtensionObject(
+        a: *mut PyObject,
+        b: *mut PyObject,
+        c: *mut PyObject,
+        d: *mut PyObject,
+    ) -> c_int;
+    #[cfg(Py_3_7)]
+    pub fn _PyImport_FixupExtensionObject(
+        a: *mut PyObject,
+        b: *mut PyObject,
+        c: *mut PyObject,
+        d: *mut PyObject,
+    ) -> c_int;
+}
+
+#[repr(C)]
+#[derive(Copy, Clone)]
+pub struct _inittab {
+    pub name: *const c_char,
+    pub initfun: Option<unsafe extern "C" fn()>,
+}
+
+#[cfg_attr(windows, link(name = "pythonXY"))]
+extern "C" {
+    #[allow(unused)]
+    pub static mut PyImport_Inittab: *mut _inittab;
+    #[allow(unused)]
+    pub fn PyImport_ExtendInittab(newtab: *mut _inittab) -> c_int;
```

Comment:
For some reason I'm getting a compiler warning about `PyImport_ExtendInittab()` being unused, so I suppressed this. Unsure why. Maybe the presence of the `pub static mut` above triggers it?

Reply:
Yeah, that was it. I moved this to its own block 🤷‍♂️